tests/utils/test_log_handlers.py

```diff
@@ -36,6 +39,18 @@ def setUp(self):
         super(TestFileTaskLogHandler, self).setUp()
         # We use file task handler by default.
         logging.config.dictConfig(DEFAULT_LOGGING_CONFIG)
+        session = Session()
```

Comment:
We're repeating code here. Maybe we can create a separate function `cleanup`, that performs the delete operations. Furthermore, I'm a big fan of the `with create_session() as session:` syntax:
https://github.com/apache/incubator-airflow/blob/7765278479b81a73244ff44e47973bbc5bf9bcef/airflow/utils/db.py#L35

Reply:
Oh nice, I didn't know that create_session existed. Have changed as you suggested.